Makefile

```
@@ -0,0 +1,59 @@
+# Tauri Development Makefile
+# Common development tasks for the project
+
+.PHONY: help dev build clean lint format check test tauri-dev tauri-build setup all
+
+# Default target
+all: help
+
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (11).

(maxbodylength)